sample/src/Clock.fs

```diff
@@ -144,9 +144,14 @@ let styles() =
 
 [<LitElement("my-clock")>]
 let Clock() =
-    let props = LitEl.init({| hourColor = Prop "lightgreen" |}, styles())
+    let props =
+        LitEl.init<{| hourColor: string; |}>(
+            [ "hourColor", [CustomAttribute "hour-color"; InitialValue "purple"; Reflect ] ],
+            styles()
+        )
     let model, dispatch = Hook.useElmish(init, update)
-    view props.hourColor.Value model dispatch
+    JS.setTimeout (fun () -> props?("hourColor") <- "green") 2000
```

Comment:
for this particular use case it might look weird but for complex objects, reacting to property changes is expected, specially from frameworks like angular/vue, which tend to to work with properties instead of just attributes

Reply:
Are the properties expected to be modifiable internally? In that case we could make `.Value` settable so you don't need to access it dynamically (although the recommended way for asynchronous internal events are Elmish commands). But if you just want to simulate how the component reacts to property changes, would it make more sense to do it from the parent component?

Reply:
> Are the properties expected to be modifiable internally 

Not internally, I was trying to simulate an external change, components are expected to react to external changes to both properties and attributes

> would it make more sense to do it from the parent component?

Yeah `<my-clock .hourColor={color}></my-clock>`